cypress/component/delegation-accordion.cy.tsx

```diff
@@ -0,0 +1,151 @@
+import * as cypress from '@testing-library/cypress';
+
+import { ApiDelegationAccordion } from '../../src/components/ApiDelegationOverviewPage/ApiDelegationAccordion/ApiDelegationAccordion';
+
+describe('DelegationAccordion Header', () => {
+  it('should show delete button when it contains orgs that are not soft deleted', () => {
+    const orgs = [
+      { id: 1, name: 'Virksomhet 1', isSoftDelete: false },
+      { id: 2, name: 'Virksomhet 2', isSoftDelete: false },
+      { id: 4, name: 'Virksomhet 3', isSoftDelete: false },
+    ];
```

Comment:
can't comment file names so I have to put it here. Rename delegation-accordion.cy.tsx => api-delegation-accordion.cy.ts

Reply:
good catch!